examples/dendrogram.py

```diff
@@ -146,8 +148,7 @@ def _dendrogram(neuron_, show_diameters=True):
     '''Main dendrogram function
     '''
     from copy import deepcopy
-    import sys
-    sys.setrecursionlimit(3000)
+
     # neuron is copied because otherwise the tree modifications that follow
     # will be applied to our original object
     neuron = deepcopy(neuron_)
```

Comment:
Maybe we can explore copying the raw data array, then re-building the neuron. This should avoid the recursion limit problem.